src/library/modules/OpenDBSubmission.js

```diff
@@ -37,6 +38,32 @@
 				'api_req_map/start': this.processStartNext,
 				'api_req_map/next': this.processStartNext,
 
+				// battles for formation / etc
+				'api_req_sortie/battle': this.processBattle,
+				'api_req_sortie/airbattle': this.processBattle,
+				// the following two are commented out 
+				// as poi "plugin-report" doesn't seem to support them.
+				// (might have been deprecated)
+				// 'api_req_sortie/night_to_day': this.processBattle,
+				// 'api_req_battle_midnight/battle': this.processBattle,
```

Comment:
why not handle this regular night battle (following day), but those does for combined fleet?